app/views/users/show.html.erb

```diff
@@ -1,26 +1,26 @@
-<div class="user-show-c">
-    <div class="user-box">
-        <img class="user-img" src=<%= @user.photo %> alt="user"/>
-        <div class="user-info-box">
-            <h2 class="user-name"><%= @user.name %></h2>
-            <p class="user-post-count">Number of posts: <%= @user.posts_counter%></p>
-        </div>
-    </div>
-
-    <p class="user-bio"><%= @user.bio %></p>
-
-
-    <% @user.three_recent_posts.each do |post| %>
-        <%= link_to user_post_path(@user.id, post.id), class:"text-dec" do %> 
-        <div class="user-recent-post">
-            <h2 class="user-post-title">Post : <%= post.title %></h2>
-            <p class="user-post-text"><%= post.text %></p>
-            <h3 class="user-post-counters">Comments: <%= post.comments_conter%>, Likes: <%= post.likes_counter %><h3>
-        </div>
-        <% end %>
-    <% end%>
-
-    <button class="button">
-        <%= link_to "See all posts", user_posts_path(@user), class:"text-dec" %>
-    </button>
+<div class="user-show-c">
+    <div class="user-box">
+        <img class="user-img" src=<%= @user.photo %> alt="user"/>
+        <div class="user-info-box">
+            <h2 class="user-name"><%= @user.name %></h2>
+            <p class="user-post-count">Number of posts: <%= @user.posts_counter%></p>
+        </div>
+    </div>
+
+    <p class="user-bio"><%= @user.bio %></p>
+
+
+    <% @user.three_recent_posts.each do |post| %>
+        <%= link_to user_post_path(@user.id, post.id), class:"text-dec" do %> 
+        <div class="user-recent-post">
+            <h2 class="user-post-title">Post : <%= post.title %></h2>
+            <p class="user-post-text"><%= post.text %></p>
+            <h3 class="user-post-counters">Comments: <%= post.comments_conter%>, Likes: <%= post.likes_counter %><h3>
+        </div>
+        <% end %>
+    <% end%>
+
+    <button class="button">
+        <%= link_to "See all posts", user_posts_path(@user.id), class:"text-dec" %>

```

Comment:
- [ ] See all posts button is not working. Kindly fix it. OPTIONAL

![image](https://user-images.githubusercontent.com/38635158/192040537-98c77989-27f0-4caf-ad7b-5eba527ace9a.png)